Add GDC 2023 retrospective blog post (#615)
godotengine/godot-website: collections/_article/gdc-2023-retrospective.md

---
title: "GDC 2023: Retrospective"
excerpt: "We attended the Game Developers Conference (GDC) in San Francisco this year."
categories: ["events"]
author: Clay John
image: /storage/blog/covers/gdc-2023-retrospective.jpg
date: 2023-04-05 14:00:00
---

> For the first time ever Godot had a booth at the Game Developer Conference in San Francisco. In the past we never considered having a booth as the cost is totally prohibitive for us. However, this year the booth was graciously donated by W4 Games, without them, it would not have been possible to have the kind of presence we had.

Reviewer: ```suggestion
For the first time ever, Godot had a booth at the [Game Developers Conference](https://gdconf.com/) in San Francisco. In the past, we never considered having a booth as the cost is totally prohibitive for us. However, this year the booth was graciously donated by W4 Games. Without them, it would not have been possible to have the kind of presence we had.
```

Author: Just a small change to your suggestion as it includes an extra comma that shouldn't be there. Introductory phrases like "in the past" can include a comma or not as a matter of style. However, the trend these days is to not include a comma unless it is longer than 5 or so words. My personal preference is to avoid commas unless they are necessary to break up the information and provide a natural pause. 

```suggestion
For the first time ever, Godot had a booth at the [Game Developers Conference](https://gdconf.com/) in San Francisco. In the past we never considered having a booth as the cost is totally prohibitive for us. However, this year the booth was graciously donated by W4 Games. Without this huge donation, it would not have been possible to have the kind of presence we had.
```

To give you a taste of what the booth (and our experience at GDC) was like, here is a short video prepared by Emilio Coppola which summarizes our experience this year.

<iframe width="560" height="315" src="https://www.youtube.com/embed/vdAwlI6NG0I" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>

> Edited by Emi. Music track: [Builder - QUEEN JAZZ](https://queenjazz.bandcamp.com/album/bops) (Used with permission).

Reviewer: ```suggestion
Edited by Emi. Music track: [Builder - QUEEN JAZZ](https://queenjazz.bandcamp.com/album/bops) (used with permission).
```

> In this post we would like to discuss our experience from the perspective of the project and highlight what went well and what we could have improved.

Reviewer: ```suggestion
In this post, we would like to discuss our experience from the perspective of the project and highlight what went well and what we could have improved.
```

Author: As explained above, there shouldn't be a comma here

> One thing that was very clear throughout the week is that the industry is ready for an open source game engine to stand shoulder-to-shoulder with the giants in the industry and people are overwhelmingly rooting for our continued success. A special thank you to everyone who cares about the project and wants us to succeed, we do this for you.

Reviewer: ```suggestion
One thing that was very clear throughout the week is that the industry is ready for an open source game engine to stand shoulder-to-shoulder with the giants in the industry. People are overwhelmingly rooting for our continued success. A special thank you to everyone who cares about the project and wants us to succeed; we do this for you.
```

Author: ```suggestion
Two things thing became very clear to us throughout the week. The first is that the industry is ready for an open source game engine to stand shoulder-to-shoulder with the giants in the industry. The second is that people are overwhelmingly rooting for our continued success. A special thank you to everyone who cares about the project and wants us to succeed; we do this for you.
```

## What went well

> ![](/storage/blog/gdc/2023/crowd.jpg)

Reviewer: Add alt text for accessibility:

```suggestion
![Crowd at the Godot GDC stand](/storage/blog/gdc/2023/crowd.jpg)
```

**The booth:** The booth was an amazing place to hang out, chat, and play a few games. From the time the expo hall opened until it closed each day, the booth was full of people. We were very pleased to see the number of people interested in Godot and games made with Godot!

**The Godot party:** Thank you to GitHub for sponsoring the Godot Party at GitHub HQ. They were truly wonderful hosts who went above and beyond. At the party, we had dinner, Godot-themed drinks, and lovely talks from both [Nathan (GDQuest)](https://www.youtube.com/watch?v=ZRFv2rTJLUY) and [Juan (reduz)](https://www.youtube.com/watch?v=XduuHNOGGqI). The Godot Party was an excellent opportunity to mingle with other Godot community members and unwind after a busy day.

> **The community:** We had the chance to meet many Godot community members from across areas including users, educators, contributors, and moderators. Many of us have been contributing to the project for years without meeting each other face-to-face. The opportunity to put faces-to-names alone makes the trip worthwhile. 

Reviewer: ```suggestion
**The community:** We had the chance to meet many Godot community members from across areas including users, educators, contributors, and moderators. Many of us have been contributing to the project for years without meeting each other face-to-face. The opportunity to put faces to names alone makes the trip worthwhile.
```

**The pins:** We purchased small pins to give away at the booth and they were a huge hit. People loved them and we loved giving them away. We had many people track down the booth after seeing someone else displaying a Godot pin.

## What we can improve

> ![](/storage/blog/gdc/2023/gh-event.jpg)

Reviewer: ```suggestion
![Godot GitHub party](/storage/blog/gdc/2023/gh-event.jpg)
```

> **More pins:** We ordered 1000 pins to giveaway at the booth and when meeting other folks in the industry. Fortunately for us Ariel Manzur (through Ramatak) ordered 1000 more for us. The pins went fast and they drew a lot of people to the booth. Next year we would like to seek sponsors in advance specifically to buy pins (and maybe other swag).

Reviewer: ```suggestion
**More pins:** We ordered 1000 pins to giveaway at the booth and when meeting other folks in the industry. Fortunately for us, Ariel Manzur (through Ramatak) ordered 1,000 more for us. The pins went fast and they drew a lot of people to the booth. Next year, we would like to seek sponsors in advance specifically to buy pins (and maybe other swag).
```

Author: ```suggestion
**More pins:** We ordered 1000 pins to giveaway at the booth and when meeting other folks in the industry. Fortunately for us, Ariel Manzur (through Ramatak) ordered 1,000 more for the booth. The pins went fast and they drew a lot of people to the booth. Next year we would like to seek sponsors in advance specifically to buy pins (and maybe other swag).
```

> **T-shirts:** We ordered T-shirts for all the volunteers who would be staffing the booth. While not at the booth, many of us were approached by Godot users and fans who noticed the shirt. Next year we would like to have the Godot logo on the front and the back of the shirt to be seen by even more people!

Reviewer: ```suggestion
**T-shirts:** We ordered T-shirts for all the volunteers who would be staffing the booth. While not at the booth, many of us were approached by Godot users and fans who noticed the shirt. Next year, we would like to have the Godot logo on the front and the back of the shirt to be seen by even more people!
```

> **Organization of booth:** We had a central round counter in the center of the booth and a set of chairs just behind. Our hope was to simultaneously draw users in to the booth and provide a place to hang out and sit down (there isn’t a lot of seating nearby). However we found that it was very crowded around the counter from people watching and playing the games. For next year we want to make it a little easier to get to the counter. Similarly, we found that the chairs were really under utilized as the booth was constantly busy (and thus noisy!). For next year, if we do dedicated seating, we would like to separate it a bit from the mayhem. 

Reviewer: ```suggestion
**Organization of booth:** We had a central round counter in the center of the booth and a set of chairs just behind. Our hope was to simultaneously draw users in to the booth and provide a place to hang out and sit down (there isn’t a lot of seating nearby). However, we found that it was very crowded around the counter from people watching and playing the games. For next year, we want to make it a little easier to get to the counter. Similarly, we found that the chairs were really underutilized as the booth was constantly busy (and thus noisy!). For next year, if we do dedicated seating, we would like to separate it a bit from the mayhem. 
```

Author: ```suggestion
**Organization of booth:** We had a central round counter in the center of the booth and a set of chairs just behind. Our hope was to simultaneously draw users in to the booth and provide a place to hang out and sit down (there isn’t a lot of seating nearby). However, we found that it was very crowded around the counter from people watching and playing the games. Next year we want to make it a little easier to get to the counter. Similarly, we found that the chairs were really underutilized as the booth was constantly busy (and thus noisy!). For next year, if we do dedicated seating, we would like to separate it a bit from the mayhem.
```

**Business cards:** We had pins to give away but we didn’t have anything with the Godot website or information about Godot. Many people wanted a card or pamphlet which would give them something to look up once they got home. We would like to have small business cards with a link to the website and perhaps a QR code to download the engine.

> **Organize more hangouts:** This was a first GDC for most of us and we were pretty overwhelmed organizing things (again thank you to W4 who handled a lot of the organization as well, we couldn’t have done this without you). Once the week ended and the dust settled we found ourselves wishing we had organized a dinner or hangout with the booth staff and game exhibitors. Despite spending all day within 10 feet of some people we barely got a chance to talk with everyone at the booth. In the future, we would like to schedule an event to acquaint the booth staff with the exhibitors showing off their games.

Reviewer: ```suggestion
**Organize more hangouts:** This was a first GDC for most of us and we were pretty overwhelmed organizing things (again thank you to W4 who handled a lot of the organization as well, we couldn’t have done this without you). Once the week ended and the dust settled, we found ourselves wishing we had organized a dinner or hangout with the booth staff and game exhibitors. Despite spending all day within 10 feet of some people, we barely got a chance to talk with everyone at the booth. In the future, we would like to schedule an event to acquaint the booth staff with the exhibitors showing off their games.
```

> **Show off the engine more:** At the booth we had 8 TVs with playable Godot games as well as 1 big TV with the 2022 showreels on loop. While this made for an impressive display, it didn’t show people what Godot is, many people passing by asked if we were a publisher or a game platform. We would like to utilize the space to show off the editor and the game creation process more, either with live demos or a prerecorded video.

Reviewer: ```suggestion
**Show off the engine more:** At the booth, we had 8 TVs with playable Godot games as well as 1 big TV with the 2022 showreels on loop. While this made for an impressive display, it didn’t show people what Godot is, many people passing by asked if we were a publisher or a game platform. We would like to utilize the space to show off the editor and the game creation process more, either with live demos or a prerecorded video.
```

## Final thoughts

> ![](/storage/blog/gdc/2023/booth.jpg)

Reviewer: ```suggestion
![Godot GDC booth](/storage/blog/gdc/2023/booth.jpg)
```

> Overall, GDC was fun, exhausting, and productive. We have been given a lot think about and discuss as we look forward to the next year of development. In the coming months expect some exciting announcements as well as more detailed plans for development of Godot 3.6 and 4.1.

Reviewer: ```suggestion
Overall, GDC was fun, exhausting, and productive. We have been given a lot think about and discuss as we look forward to the next year of development. In the coming months, expect some exciting announcements as well as more detailed plans for development of Godot 3.6 and 4.1.
```

> While Godot is now starting to have a large presence in the industry, we do so on a shoestring budget. The cost to the Godot Foundation for attending GDC was only a few thousand dollars (spent on the pins, t-shirts, and reimbursements for some volunteers and exhibitors). Large events like this aren’t generally possible for us unless a company brings us along (like W4 did this year). Despite the huge amount of support we received to attend GDC we still rely on donations to fund development.

Reviewer: ```suggestion
While Godot is now starting to have a large presence in the industry, we do so on a shoestring budget. The cost to the Godot Foundation for attending GDC was only a few thousand dollars (spent on the pins, t-shirts, and reimbursements for some volunteers and exhibitors). Large events like this aren’t generally possible for us unless a company brings us along (like W4 did this year). Despite the huge amount of support we received to attend GDC, we still rely on donations to fund development.
```
